File: src/JavaScript/easy/string/problem2678.js
/**
 * @param {string[]} details
 * @return {number}
 */


var countSeniors = function(details) {
    return details.map(detail => detail.slice(-4,-2) * 1).filter(age => age > 60).length
};


let details = ["7868190130M7522","5303914400F9211","9273338290F4010"]

countSeniors(details)

File: src/Python/easy/string/problem2678.py
from typing import List

class Solution:
    def countSeniors(self, details: List[str]) -> int:
        return len(list(filter(lambda x:x > 60, list(map(lambda x: int(x[-4:-2]), details)))))


details = ["7868190130M7522","5303914400F9211","9273338290F4010"]

solution = Solution()
solution.countSeniors(details)